db/migrate/20190408004639_add_avail_sun_am_to_volunteers.rb

```diff
@@ -0,0 +1,5 @@
+class AddAvailSunAmToVolunteers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :volunteers, :avail_sun_am, :string
```

Comment:
The type should be a boolean instead of a string -- but this is just a suggestion, you don't need to make this change now.

Reply:
Ok, I attempted to change to boolean, but couldn't get my form to save and show the results. I just need to learn how to use booleans in form helpers.

Reply:
I changed it back to string. It works fine for now.